src/pages/TCY/components/Stake/Stake.tsx

```diff
@@ -95,77 +75,6 @@ export const StakeRoutes: React.FC<TCYRouteProps & { activeAccountNumber: number
   activeAccountNumber,
 }) => {
   const location = useLocation()
-  const [stakeTxid, setStakeTxid] = useState<string>()
-  const queryClient = useQueryClient()
-  const dispatch = useAppDispatch()
-  const { getValues } = useFormContext<StakeFormValues>()
-  const { isDrawerOpen, openActionCenter } = useActionCenterContext()
-
-  const toast = useNotificationToast({
-    duration: isDrawerOpen ? 5000 : null,
-  })
-
-  const accountId = useAppSelector(state => {
-    const accountIdsByAccountNumberAndChainId = selectAccountIdByAccountNumberAndChainId(state)
-    const accountNumberAccounts = accountIdsByAccountNumberAndChainId[activeAccountNumber]
-    return accountNumberAccounts?.[thorchainChainId] ?? ''
-  })
-
-  const handleTxConfirmed = useCallback(async () => {
-    if (!stakeTxid) throw new Error('Stake Txid is required')
-
-    const amountCryptoPrecision = bnOrZero(getValues('amountCryptoPrecision')).toFixed()
-
-    dispatch(
-      actionSlice.actions.upsertAction({
-        id: stakeTxid,
-        type: ActionType.Deposit,
-        transactionMetadata: {
-          displayType: GenericTransactionDisplayType.TCY,
-          txHash: stakeTxid,
-          chainId: thorchainChainId,
-          accountId,
-          assetId: tcyAssetId,
-          amountCryptoPrecision,
-          message: 'RFOX.stakeSuccess',
-        },
-        status: ActionStatus.Complete,
-        createdAt: Date.now(),
-        updatedAt: Date.now(),
-      }),
-    )
-
-    toast({
-      id: stakeTxid,
-      duration: isDrawerOpen ? 5000 : null,
-      status: 'success',
-      render: ({ onClose, ...props }) => {
-        const handleClick = () => {
-          onClose()
-          openActionCenter()
-        }
-        return (
-          <GenericTransactionNotification
-            // eslint-disable-next-line react-memo/require-usememo
-            handleClick={handleClick}
-            actionId={stakeTxid}
-            onClose={onClose}
-            {...props}
-          />
-        )
-      },
-    })
-    await queryClient.invalidateQueries({ queryKey: ['tcy-staker'] })
```

Comment:
q: where did this move?

Reply:
Assuming you took https://github.com/shapeshift/web/pull/10066 as a reference, it worked because we were already invalidating the relevant query in `useGenericTransactionSubscriber()` - now the only invalidation still happening is in `handleTxConfirmed` in `<ClaimRoutes />`.
Meaning that stake/unstake Txs won't update the TCY staked balance anymore

Reply:
>  We need something that automatically will refetch when they land on or are already on the TCY page and the number of successful staking transactions has changed since last fetch.

I'm confused @premiumjibles.

What's the issue re: doing it the exact same way as other places i.e invalidating on the hook itself? Invalidation should still be the primary way to invalidate, not sure using the `queryKey` to refetch is something we want (invalidated data for a given query key because of query key changes and manually invalidated query key are two very different things)

Reply:
Going to merge since this isn't functionally blocking, but yeah not the biggest fan of lack of consistency here - mostly an opinion of mine though so no reason to block merge for that